nix/tests/sql/pgmq_data_api_wrapper.sql

```diff
@@ -0,0 +1,465 @@
+/*	
+    This test is to validate the SQL for the Supabase Queues integration that
+	will be triggered in the FE and documented for how to manually expose
+	queues over supabase client libs by wrapping `pgmq`'s functions into a
+	separate schema that we can add to PostgREST's 'exposed_schemas' setting
+*/
+
+/*
+	Emulate Role Setup from migrations/db/init-scripts/00000000000000-initial-schema.sql#L5
+*/
+
+-- Supabase super admin
+alter user  supabase_admin with superuser createdb createrole replication bypassrls;
+
+-- Supabase replication user
+create user supabase_replication_admin with login replication;
```

Comment:
```suggestion
create user supabase_replication_admin with replication;
```
or
```suggestion
create role supabase_replication_admin with login replication;
```

`create user` is an alias for `create role with login`